### DISCUSSION TITLE
Milky Way

### DISCUSSION THREAD

**Discussion by relaxcourageous** [Stellarium/stellarium#2336] [Q&A]
Does Stellarium take into account the shifting of the Milky Way? (relative to other constellations) i.e. Argo Navis

**Comment by gzotti** [comment]
What shifting do you mean?

**Reply by relaxcourageous** [reply]
At one time the Argo Navis appeared to be skimming on the Milky Way, it has now "come off". This I assume would be due to the motion of the Milky Way relative to the constellations. https://en.wikipedia.org/wiki/Argo_Navis "Some stars of Puppis and Vela can be seen from [Mediterranean](https://en.wikipedia.org/wiki/Mediterranean_Sea) latitudes in winter and spring, the ship appearing to skim along the "river of the [Milky Way](https://en.wikipedia.org/wiki/Milky_Way)."[[2]](https://en.wikipedia.org/wiki/Argo_Navis#cite_note-Massey-2) Due to [precession of the equinoxes](https://en.wikipedia.org/wiki/Precession), the position of the stars from Earth's viewpoint has shifted southward, and though most of the constellation was visible in Classical times, the constellation is now not easily visible from most of the northern hemisphere.[[3]](https://en.wikipedia.org/wiki/Argo_Navis#cite_note-3)"

**Reply by gzotti** [reply]
OK. It's not the Milky Way, but Earth's axis that shifts. This is precession. By this process, all stars shift. Given that the Milky Way is simply the mass of stars too dim and remote to see them as individual points of light, this shift also affects the Milky Way in the same way. There is no shift of the Milky Way relative to the constellations.

**Comment by JWRP** [comment]
Hi, I am interested in the Milky Way Core and where it is visible during the year. As far as I can read there is a "Milky Way Season" from March to October where the core is visible in he Northern Sky. From November to February it shouldn't be visible. However it doesn't seem to be the case in Stellarium, or? Also it should rise in the south east in March and in south west in October. That doesn't seem to be the case either as far as I can see..
Do I get it wrong or can someone help me out here?
Best J

**Reply by gzotti** [reply]
The position is exact. The visual impression is hard to properly simulate and depends on atmospheric conditions, esp. light pollution.
The Milky Way center always rises in the same point on the horizon, dictated by its declination and geographical latitude of observer. And of course everything rises in the eastern half, and sets in the western half. From central Europe, the Milky Way centre is only visible low in the south at best, never in the northern sky.

**Reply by JWRP** [reply]
Hi gzotti. Thanks so much for your reply. As I understand your reply the Milky Way Core rises and sets in the same locations all year (SE to SW), in regard to the location the person observing is positioned and declination. However, when I am reading this article, the writer seems to be of another opinion, which confuses me. https://capturetheatlas.com/best-time-to-see-the-milky-way/#northern
Best J

**Reply by gzotti** [reply]
The first half seems OK, but I will not read all of this long text. He gives extensive discussion about latitude-dependent visibility in relation to seasons. What is the actual problem?

**Reply by JWRP** [reply]
I am interested in finding out where and when the Milky Way Core is visible from different locations. I get a little confused when reading the article and afterwards looking at Stellarium, but perhaps it is just my settings for viewing. The illumination is important. The latitude decides how high the core gets in the sky. How does the seasons play a role?

**Reply by 10110111** [reply]
The season defines where the Sun is relative to the Milky Way. If the Sun is roughly in the same direction as the Milky Way core, then you'll not be able to see the core at the given part of year, since atmosphere will scatter much more light towards you than the Milky Way emits.

**Reply by JWRP** [reply]
Thank you so much for the answers! I will explore this further. Best Jens
